### PR TITLE
create a backup script with python and selenium

### PR DESCRIPTION
**Pull request by rebeca53**
The script is the first version. It opens the folder link in a browser, clicks the button to download it, and waits 20 seconds. Then close the browser.

"It works on my machine"

### CODE REVIEW

File: tools/backupsecure.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# Authenticate and create PyDrive client
+gauth = GoogleAuth()
+gauth.DEFAULT_SETTINGS['client_config_file'] = 'client_secrets.json'
+gauth.DEFAULT_SETTINGS['oauth_scope'] = [
+    'https://www.googleapis.com/auth/drive',
+    'https://www.googleapis.com/auth/drive.file',
+    'https://www.googleapis.com/auth/drive.metadata'
+]
+gauth.DEFAULT_SETTINGS['client_id'] = '892748311642-uive10rq7nsi55ajiu42r0tpkpaftfp6.apps.googleusercontent.com'
+gauth.DEFAULT_SETTINGS['client_secret'] = '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'
```
Comment (geomunduswebteam) [inline tools/backupsecure.py:31]: Is this sensitive information?

File: tools/backupsecure.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+from pydrive.auth import GoogleAuth
+from pydrive.drive import GoogleDrive
+import os
+from zipfile import ZipFile
+import mimetypes
+
```
Comment (geomunduswebteam) [inline tools/backupsecure.py:6]: Considering that we have this script using the google auth, do we still need to have the backup with selenium?

Comment (chubacha) [reply tools/backupsecure.py:6]: Yes. The selenium script is the only one working. However, selenium can't do the folder that needs authentication, which is where the authentication script comes in. However that one is not working yet. It downloads the folders and files, but as empty files, with zero bytes.

Comment (chubacha) [reply tools/backupsecure.py:6]: Well, selenium may be able to do the one with authentication, but we haven't figured it out yet

File: tools/backupsecure.py (hunk 1 above)
Comment (chubacha) [reply tools/backupsecure.py:31]: Yes

**Pull request merged**